- add 'log2()' and 'log2f()' declarations
- this patch is only needed for FreeBSD libc versions < 9

+++ src/lib/libc/lib/msun/src/math.h
@@ -204,6 +204,7 @@
 double	ldexp(double, int);
 double	log(double);
 double	log10(double);
+double	log2(double);
 double	modf(double, double *);	/* fundamentally !__pure2 */
 
 double	pow(double, double);
@@ -317,6 +318,7 @@
 int	ilogbf(float) __pure2;
 float	ldexpf(float, int);
 float	log10f(float);
+float	log2f(float);
 float	log1pf(float);
 float	logf(float);
 float	modff(float, float *);	/* fundamentally !__pure2 */
